README.md

```diff
@@ -25,7 +25,7 @@
 
 # 📖 [MEME-GALAXY] <a name="about-project"></a>
 
-**Meme Galaxy** is a single-page app built using React JS and Vite whose aim is to display different memes with differenct categories accodrding to data fetched from <b>imgflip API.</b> Further information is available in the <a href="https://drive.google.com/file/d/1eZNRTECZUpxfykamc0erwobgSFsP5tY5/view?usp=sharing">video presentation</a>
+**Meme Galaxy** is a single-page app built using React JS and Vite whose aim is to display different memes with differenct categories accodrding to data fetched from <b>imgflip API.</b> For more information visit the <a href="https://drive.google.com/file/d/1eZNRTECZUpxfykamc0erwobgSFsP5tY5/view?usp=sharing">video presentation</a>
 ## 🛠 Built With <a name="built-with"> React JS</a>
```

Comment:
- [ ] Also, In this project, `eslint-disabler` is not allowed to use. I was not able to point out all of them but, this is a reference for you. Please disable all of them. Thank you!

<img width="436" alt="Screenshot 2023-05-26 at 22 51 03" src="https://github.com/MarwanDev/Meme-Galaxy/assets/69011963/1c58b0b7-0658-4308-89d7-8557f2c872c7">

> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.